lib/storekit/receipt.rb

```diff
@@ -1,19 +1,42 @@
+require 'ostruct'
+
 module StoreKit
-  class Receipt
-    attr_reader :receipt_data
+  class Receipt < OpenStruct
+    TRANSFORM = {
+       'unique_identifier' => true,
+       'product_id' => true,
+       'bid' => true,
+       'unique_vendor_identifier' => true,
+       'bvrs' => true,
+       'transaction_id' => proc { |v| v.to_i },
+       'original_transaction_id' => proc { |v| v.to_i },
+       'quantity' => proc { |v| v.to_i },
+       'item_id' => proc { |v| v.to_i },
+       'web_order_line_item_id' => proc { |v| v.to_i },
```

Comment:
The ID -> Integer transformations are invalid. According to the [official documentation on receipt fields][1], these are strings.

[1]: https://developer.apple.com/library/content/releasenotes/General/ValidateAppStoreReceipt/Chapters/ReceiptFields.html#//apple_ref/doc/uid/TP40010573-CH106-SW11